sources/packages/forms/src/form-definitions/studentprofile.json

```diff
@@ -3597,7 +3597,7 @@
           "tags": [],
           "properties": {},
           "conditional": {
-            "show": null,
+            "show": "",
```

Comment:
On create student , the declaration is still not modified. 
![image](https://github.com/user-attachments/assets/fca9cd01-059c-41df-ae99-d747952fc786)

Reply:
have updated the name across files now